tests/unit/FileTest.php

```diff
@@ -20,7 +20,10 @@ public function newClient(): void
 		$api_key = getenv('API_KEY');
 		$reference_id = getenv('REFERENCE_ID');
 		$bucketId = 'test-bucket';
-		$this->client = new  \Supabase\Storage\StorageFile($api_key, $reference_id, $bucketId);
+		$scheme = 'https';
```

Comment:
After the __constructor is updated none of these should be needed - please remove them.